ui/apps/platform/cypress/integration/policies/policiesTable.test.js

```diff
@@ -286,7 +286,7 @@ describe('Policies table', () => {
         cy.intercept('POST', `${api.policies.policies}?enableStrictValidation=true`).as(
             'POST_policies'
         );
-        cy.get(selectors.wizardBtns.step7).click();
+        cy.get(selectors.wizardBtns.step6).click();
```

Comment:
An observation for whenever we next invest in integration tests.

Although there would be some code change any way we organize it:
1. Move and rename policies.js from helpers to policies/Policies/helpers.js
2. Encapsulate selector in something like `goToStep6` helper function.
3. Encapsulate the next 2 lines in something like `savePolicy` helper.